client/my-sites/upgrades/domain-management/transfer/transfer-to-other-user/index.jsx

```diff
@@ -0,0 +1,218 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { includes, head, omit, find } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { localize } from 'i18n-calypso';
+import Card from 'components/card';
+import { getCurrentUser } from 'state/current-user/selectors';
+import Header from 'my-sites/upgrades/domain-management/components/header';
+import Main from 'components/main';
+import paths from 'my-sites/upgrades/paths';
+import FormSelect from 'components/forms/form-select';
+import FormButton from 'components/forms/form-button';
+import FormFieldset from 'components/forms/form-fieldset';
+import wp from 'lib/wp';
+import { getSelectedDomain } from 'lib/domains';
+import NonOwnerCard from 'my-sites/upgrades/domain-management/components/domain/non-owner-card';
+import DomainMainPlaceholder from 'my-sites/upgrades/domain-management/components/domain/main-placeholder';
+import SectionHeader from 'components/section-header';
+import Dialog from 'components/dialog';
+import { successNotice, errorNotice } from 'state/notices/actions';
+import page from 'page';
```

Comment:
What is it with external dependencies trying to sneak into the end of internal ones? Is it a new fad?